packages/params/tsconfig.doc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {},
+  // `typescript-docs-verifier` uses `ts-node` to compile README snippets
```

Comment:
If you want to add comments in JSON, rename those to `.jsoc` extension so it does not highlight as error.

Reply:
that's really only highlighted this way in the diff, we have comments in other tsconfig files as well

https://github.com/ChainSafe/lodestar/blob/14855ea72fa7c46c984ed13078aac1c21858b993/tsconfig.json#L12-L15

tsconfig itself supports comments as is, might be able to remove this if upstream issue is fixed